serde/src/private/de.rs

```diff
@@ -191,17 +191,20 @@ where
 pub mod size_hint {
     use lib::*;
 
+    #[inline]
     pub fn from_bounds<I>(iter: &I) -> Option<usize>
```

Comment:
I don't think placing `#[inline]` on generic methods does anything.

Reply:
Yeah that would make sense, since code generation has to be pushed to the caller anyways.

Reply:
I wouldn't worry about an extra inline as we certainly do the same on lots of other generic methods. If rustc or clippy ever starts to warn, we will clean them up.